src/Popover/Popover.js

```diff
@@ -240,7 +241,11 @@ class Popover extends Component {
     };
 
     a.right = rect.right || a.left + a.width;
-    a.bottom = rect.bottom || a.top + a.height;
+    if (isIOS() && document.activeElement.tagName === 'INPUT') {
```

Comment:
Side node, with the `next` branch we would be using [activeElement](https://github.com/react-bootstrap/dom-helpers/blob/master/src/activeElement.js)